R/scale-brewer.r

```diff
@@ -104,6 +104,27 @@ scale_fill_distiller <- function(..., type = "seq", palette = 1, direction = -1,
     gradient_n_pal(brewer_pal(type, palette, direction)(7), values, space), na.value = na.value, guide = guide, ...)
 }
 
+#' @export
+#' @rdname scale_brewer
+scale_colour_blender <- function(..., type = "seq", palette = 1, direction = -1, na.value = "grey50", guide = "coloursteps", aesthetics = "colour") {
```

Comment:
Why blender?

Reply:
If we have a brewer (discrete, beer) and a distiller (continuous, wine), it feels like the obvious name would be vintner.

Reply:
blender for mixed drinks 🙂 I'm open to another naming honestly. I just wanted to continue this weird tradition

Reply:
But "brewer" is the last name of Cynthia Brewer who developed the color palettes. So maybe `scale_color_pedersen()`? On a more serious note, I prefer more informative naming schemes, for example as implemented in the [colorspace ggplot2 scales.](http://colorspace.r-forge.r-project.org/articles/ggplot2_color_scales.html) Would `scale_color_brewer_binned()` or  `scale_color_brewer_b()` be so terrible? Similarly, we already have `scale_color_viridis_c()` and `scale_color_viridis_d()`, so adding `scale_color_viridis_b()` for binned would be consistent with that scheme.

Reply:
I only went this way because of `distiller` and thought it would be a nice joke to continue. I must say that really do not like the look of the `_d|c|b` suffix, so if anything better is available I will always choose that

Reply:
for the scales that already have a `_d` version then a `_b` is an obvious choice but otherwise I'm forcefully against :-)

Reply:
I think we're slightly stuck with the overly clever names for now. I'd prefer vintner over mixer, to be consistent with "people" rather than objects.

Reply:
I can see the argument for it being a person but vintner is just super obscure (or maybe I’m just not well educated). Would “bartender” be too on-the-nose?

Reply:
I feel the same. If I saw `scale_color_vintner()` I'd have no idea what it does and I probably wouldn't associate it with `scale_color_brewer()` either. I was just thinking along the same lines as Thomas. What about `scale_color_tender()`?

Reply:
Actually, I’m warming up to it

Reply:
+1 for tender

Reply:
One other thought: I can envision somebody brewing colors, and I can also envision somebody distilling discrete colors into a continuum. If we stay with the metaphor of a person working the colors, something like `scale_color_smith()` or `scale_color_plumber()` might not be bad either.

Reply:
I think tender is equally too far from a person who makes drinks — it's not a standard shortening of bar tender. How about `scale_brewer_fermenter()`?

Reply:
We have now settled on `fermenter` — I expect the perfect name will show itself to us after release 😛